docs/src/main/sphinx/admin/properties-http-client.rst

```diff
@@ -0,0 +1,258 @@
+======================
+HTTP client properties
+======================
+
+HTTP client properties allow you to configure the connection from Trino to
+external services using HTTP.
+
+The following properties can be used after adding the specific prefix to the
+property. For example, for :doc:`/security/oauth2`, you can enable HTTP/2 for
```

Comment:
We should remove all mention of HTTP/2 since it has known bugs (which is why we don't enable by default). A better example would be a custom truststore or proxy configuration, which are the common things people want to enable.